source/db/zh-CN-projects.json

```diff
@@ -29,13 +29,13 @@
   },
   {
     "development_stage": "experimental",
-    "description": "基于开源安卓项目，帮助用户检测和避开伪基站（IMSI 捕捉器）或其他无加密/弱加密基站。本项目旨在协助用户了解加密和其他一些保护机制是否处于启用状态。事物总是在发展，我们永远欢迎测试者以及热衷信息安全的开发者。不必拘束，你可以做任何你能做的贡献。我们的 [GitHub](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector)。我们的 [XDA](https://forum.xda-developers.com/showthread.php?t=1422969)。感谢你的参与！",
+    "description": "基于开源安卓项目，帮助用户检测和避开伪基站（IMSI捕捉器）或其他无加密/弱加密基站。本项目旨在协助用户了解加密和其他一些保护机制是否处于启用状态。事物总是在发展，我们永远欢迎测试志愿者以及关注信息安全的程序员志士。不必害羞，你可以做任何你能做的贡献。我们的Github：https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector。我们的XDA：https://forum.xda-developers.com/showthread.php?t=1422969。谢谢你们的参与！",
```

Comment:
This entire file seems to have gotten mangled. An update to the Chinese localizations got merged; maybe you just need to rebase or something?